src/ape/utils/abi.py

```diff
@@ -90,7 +90,7 @@ def parse(self, output_types: List[ABIType], values: Union[List, Tuple]) -> Any:
             return values
 
         elif has_tuple_array_return:
-            output_type = output_types[0]
+            output_type = output_types[0].copy(deep=True)
```

Comment:
We can also disable mutation from ABIType in `ethpm-types` to help prevent this from happening again. then, we will need to use object initialization or parsing here

Reply:
OK, switched to the object initialization strategy!